src/gdash/scripts/_contains/_contains.gml

```diff
@@ -15,9 +15,10 @@ _contains("hello", "ello");
 // => true
 */
 
-var collection = argument[0];
-var target = argument[1]
-var fromIndex = 0;
+var 

```

Comment:
Wondering, is there an actual performance boost by only `var`ing once? Also, looks like line 19 has a semicolon where it should have a comma.

Stylistically, to be in lockstep with the rest of the repo, I'd prefer discrete `var`s on each line, but if there is a provable performance gain, I'm all for it.